### ISSUE TITLE
Show more info before downloading a file

### ISSUE THREAD

**Issue by gudarzi** [gudarzi/SaveHere#21]
Show more info about a file, right after adding a new link to the queue. Also, show errors if the server can't access the file. Things to consider:
- file size (otherwise show proper message)
- chosen filename
- chosen file extension
- whether the file is protected by user:password
- any errors or exceptions

**Comment by sami0505** [comment]
Hello, could I work on this issue?

**Comment by gudarzi** [comment]
@sami0505 Hi sami. If you're interested in this project and want to help with fixing this issue, I'd be happy to assign it to you. Do you want to work on this issue?

**Comment by sami0505** [comment]
Yeah, of course! Might take me a week or so to familiarise myself with the codebase and fix the issue. If that's fine with you, I'd be eager to!

**Comment by mohammedaljader** [comment]
Hey Ehsan! I spoke to you about a month ago regarding this issue, and I'm still working on it with some progress made. Could you please assign it to me? Also, sorry @sami0505, I need to work on this issue because of my school project.

**Comment by sami0505** [comment]
Don't worry, that's alright! I got unexpectedly busy, feel free to take it!

**Comment by mohammedaljader** [comment]
Thank you @sami0505 and good luck!

**Comment by mohammedaljader** [comment]
@gudarzi I have a question regarding Direct Link Downloader. Do I need to add info button to it. If yes, how does the download from direct link work?

**Comment by gudarzi** [comment]
@mohammedaljader Yes, the direct downloader needs the info button. You can use a MudButtonGroup to put the Add and Info button in 1 place. Check out the MudBlazor docs for MudButtonGroup.

**Comment by mohammedaljader** [comment]
@gudarzi  I updated the code based on your remark and made a PR. Could you please have a look?

> [@mohammedaljader](https://github.com/mohammedaljader) Yes, the direct downloader needs the info button. You can use a MudButtonGroup to put the Add and Info button in 1 place. Check out the MudBlazor docs for MudButtonGroup.
